Update registration API with build num (#3888)
cloudflare/cloudflare-docs: content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md

- IPv4 API Endpoint: `162.159.192.1`

{{<Aside type="note">}}

> The Orchestration API endpoint is currently shared with our Consumer WARP client. Starting with build 2022.4.x we will be splitting this out to its own dedicated IP addresses.

Reviewer: ```suggestion
The Orchestration API endpoint is currently shared with our Consumer WARP client. Starting with build 2022.4.x, Zero Trust WARP and Consumer WARP will have their own dedicated API endpoints.
```

Those IP addresses will be:

> 

Author: They are all for Zero Trust
